nengo_viz/static/viz.js

```diff
@@ -378,6 +378,12 @@ VIZ.DataStore.prototype.get_shown_data = function() {
     return shown;
 }
 
+VIZ.DataStore.prototype.is_at_end = function() {
+    var ts = this.sim.time_slider;
+    return (ts.last_time < ts.first_shown_time + ts.shown_time + 0.01);
```

Comment:
Yup, that's for float fudge factor, but I made it way too large...   It does have to be a ```+```, though (with a - sign, the function can never return true).  +0.0000001 seems to work, though.